CHANGELOG.md

```diff
@@ -28,26 +34,29 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Bumped Cairo version to `2.7.0`
 - Max steps in tests (configured by argument `--max-n-steps`) now defaults to 10 million
-if not provided (changed from 4 million).
+  if not provided (changed from 4 million).
```

Comment:
Can you revert those (unwanted) formatting changes? I'd rather modify this file incrementally